src/test/acceptance/features/navigation.feature

```diff
@@ -0,0 +1,52 @@
+Feature: Application process navigation is controlled
+  In order to facilitate the beta HTBHF programme
+  As a potential claimant
+  I want to make sure that only the correct application flow can be used
+
+  Scenario Outline: Enter name page is always the first page
+    Given I am starting a new application
+    When I navigate to the <page> page
+    Then I am shown the enter name page
+    Examples:
+      | page                     |
+      | enter name               |
+      | enter national insurance |
+      | enter date of birth      |
+      | are you pregnant         |
+      | card address             |
+
+  Scenario Outline: Navigation after completing application returns to confirm page
+    Given I have completed my application
+    When I navigate to the <page> page
+    Then I am shown a successful confirmation page
```

Comment:
We will need some way of allowing applicants to clear their session in this scenario (to support multiple applicants on a shared machine). I'm starting to wonder whether instead we clear their session if they try to go back to other pages?

Reply:
I realise that this meets the requirement as stated in the story, mind.